src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -2713,24 +2636,20 @@ Real64 GLHEVert::calcBHGroutResistance(EnergyPlusData &state)
 Real64 GLHEVert::calcHXResistance(EnergyPlusData &state)
 {
     // Calculates the effective thermal resistance of the borehole assuming a uniform heat flux.
-
     // Javed, S. & Spitler, J.D. Calculation of Borehole Thermal Resistance. In 'Advances in
     // Ground-Source Heat Pump Systems,' pp. 84. Rees, S.J. ed. Cambridge, MA. Elsevier Ltd. 2016.
-
     // Eq: 3-67
 
-    using FluidProperties::GetSpecificHeatGlycol;
-
-    constexpr const char *RoutineName("calcBHResistance");
-
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:2724]:(style),[variableScope],The scope of the variable 'RoutineName' can be reduced.